Modules/Quotas/test/Quotas.Application.Tests/Metrics/UsedFileStorageSpaceMetricCalculatorTests.cs

```diff
@@ -10,27 +11,28 @@ public class UsedFileStorageSpaceMetricCalculatorTests
     public async Task Aggregate_Used_Space_Throws_OverflowException_When_Sum_Wont_Fit_In_A_Long()
     {
         // Arrange
-        var repositoryStub = new FileMetadataRepositoryStub(new List<FileMetadata>() { new() { CipherSize = 9223372036854775806 }, new() { CipherSize = 9223372036854775806 } });
+        var repositoryStub = new FileMetadataRepositoryStub(long.MaxValue);
+        var metricCalculator = new UsedFileStorageSpaceMetricCalculator(repositoryStub);
 
         // Act
-        var acting = async () => await repositoryStub.AggregateUsedSpace("", DateTime.UtcNow, DateTime.UtcNow, CancellationToken.None);
+        var acting = async () => await metricCalculator.CalculateUsage(DateTime.UtcNow, DateTime.UtcNow, "", CancellationToken.None);
```

Comment:
The empty string is confusing when reading the test.

```suggestion
        var acting = async () => await metricCalculator.CalculateUsage(DateTime.UtcNow, DateTime.UtcNow, "some-address", CancellationToken.None);
```